google-cloud-pubsub/google-cloud-pubsub.gemspec

```diff
@@ -20,6 +20,7 @@ Gem::Specification.new do |gem|
 
   gem.add_dependency "google-cloud-core", "~> 1.2"
   gem.add_dependency "google-gax", "~> 1.3"
+  gem.add_dependency "googleapis-common-protos", ">= 1.3.9", "< 2.0"
```

Comment:
```suggestion
  gem.add_dependency "googleapis-common-protos", [">= 1.3.9", "< 2.0"]
```

Reply:
Is the explicit array necessary and common? Examples in [guides.rubygems.org](https://guides.rubygems.org/patterns/#declaring-dependencies) do not show the array.

Reply:
I thought this was why the build failed.